modules/kubernetes-addons/kyverno/locals.tf

```diff
@@ -0,0 +1,64 @@
+locals {
+  kyverno_chart_name          = "kyverno"
+  kyverno_policies_chart_name = "kyverno-policies"
+  kyverno_ui_chart_name       = "policy-reporter"
+
+  default_kyverno_values          = [templatefile("${path.module}/kyverno-values.yaml", {})]
+  default_kyverno_policies_values = [templatefile("${path.module}/kyverno-policies-values.yaml", {})]
+  default_kyverno_ui_values       = [templatefile("${path.module}/kyverno-ui-values.yaml", {})]
+
+  default_kyverno_controller_helm_config = {
+    name        = local.kyverno_chart_name
+    chart       = local.kyverno_chart_name
+    repository  = "https://kyverno.github.io/kyverno/"
+    version     = "v2.5.2"
+    namespace   = "kyverno"
```

Comment:
Create a new local variable for kyverno_namespace and replace all three occurrences

Reply:
Thanks. Fixed in the recent commit.